src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2172,16 +2311,64 @@
           short-summary: Name or ID of the VM to target.
         - name: --filters
           short-summary: JSON encoded list of packet filters. Use `@<file path>` to load from file.
+    examples:
+        - name: Create a packet capture session on a VM
+          text: az network watcher packet-capture create -g {resoureceurceGroupName} -n packetCaptureName --vm {vmName} --storage-account gwteststorage123abc
```

Comment:
The usage of `{...}` to refer to a user-supplied value is different from other usage in this helpfile, and throughout the CLI. We need to standardize on either using default naming conventions and document them, __OR__ use placeholders such as these.

Reply:
Also, that's a pretty epic typo for `resoureceurceGroupName`

Reply:
Addressed.